XenModel/XenSearch/GroupAlg.cs

```diff
@@ -316,6 +332,8 @@ public override bool Populate(IAcceptGroups adapter, int indent, bool defaultExp
 
         public override void PopulateFor(IAcceptGroups adapter, GroupKey group, int indent, bool defaultExpand)
         {
+            //System.Console.WriteLine(group.GetHashCode().ToString() + "   " + group.key.ToString() );

```

Comment:
The new code is good, but I suggest we lose the (already commented-out) WriteLine.